source/content.css

```diff
@@ -817,6 +817,10 @@ a.tabnav-extra[href$='mastering-markdown/'] {
 	vertical-align: middle;
 }
 
+.rgh-fade-in {
```

Comment:
Opening bracket, but no closing bracket.

Reply:
?

Closing bracket is on 822

Reply:
Oh sorry, I had an userscript that hided that line. 😊

Reply:
That's weird, is that a feature?

Reply:
https://github.com/Mottie/GitHub-userscripts/wiki/GitHub-remove-diff-signs :)

Reply:
Refined GitHub already has that feature.

Reply:
I noticed. Refined GitHub makes them unselectable. The previously mentioned UserScript completely removed them.  Currently in the progress of disabling more conflicting UserScripts 😄 

Sorry for spamming.